src/main/java/com/getindata/connectors/http/internal/table/lookup/JavaNetHttpPollingClient.java

```diff
@@ -27,6 +27,10 @@
 @Slf4j
 public class JavaNetHttpPollingClient implements PollingClient<RowData> {
 
+    public static final String DEFAULT_REQUEST_MAX_RETRIES = "3";
```

Comment:
should this not be an int?

Reply:
We tend keep all properties as Strings, becase they are passed as Properties<String,String> from Table factories.

Later we use them like this:

```
this.httpRequestMaxRetries = Integer.parseInt(
            options.getProperties().getProperty(
                HttpConnectorConfigConstants.LOOKUP_HTTP_MAX_RETRIES,
                DEFAULT_REQUEST_MAX_RETRIES
            )
        );
```